<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinAddContact</class>
 <widget class="QDialog" name="WinAddContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>186</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>186</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: white;
font-family: Calibri;
}

QLineEdit{
background-color: transparent;
border: none;
border-bottom: 1px solid #9e9e9e;
border-radius: 0;
outline: none;
height: 3rem;
width: 100%;
margin: 0;
}
QLineEdit:focus{
    border-bottom: 3px solid #b71c1c;
}

QLineEdit:disabled {
    background-color: #e57373;
	color: white;
}

QPushButton{
    background-color: #b71c1c;
	border: solid 1px;
	color: white;
}
QPushButton:pressed {
    background-color: #791010;
    border-style: inset;
}

QPushButton:disabled {
    background-color: #e57373;
	color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Contact Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TxtContactName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Contact Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TxtContactNumber">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="inputMask">
        <string>(+63\9)999999999</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="BtnAdd">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add Contact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnCancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
